product/rcar/module/rcar_ext_clock/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Arm SCP/MCP Software
+# Copyright (c) 2021, Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+add_library(${SCP_MODULE_TARGET} SCP_MODULE)
+
+target_include_directories(${SCP_MODULE_TARGET}
+                           PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include"
+                           PUBLIC "${CMAKE_SOURCE_DIR}/arch/arm/armv8-a/include/lib")
```

Comment:
This include directory appears to belong to the architecture target. You'll need to provide that as part of the `target_link_libraries` invocation below, rather than trying to include another libraries headers directly.

Reply:
@CJKay : Please find the fix as suggested